File: unity_app/Assets/Scripts/Core/Enums/WeatherEnums.cs
namespace EventPlannerSim.Core
{
    /// <summary>
    /// Represents the type of weather condition.
    /// </summary>
    public enum WeatherType
    {
        Clear,
        Cloudy,
        LightRain,
        HeavyRain,
        ExtremeHeat,
        ExtremeCold
    }

    /// <summary>
    /// Simplified weather risk indicator for Stage 1 display.
    /// </summary>
    public enum WeatherRisk
    {
        Good,
        Risky,
        Bad
    }
}

File: EventApp/EventsApp/Models/Enums/WeatherEnums.swift
import Foundation

enum WeatherType: String, CaseIterable, Codable, Hashable {
    case clear
    case cloudy
    case lightRain
    case heavyRain
    case extremeHeat
    case extremeCold
}

enum WeatherRisk: String, CaseIterable, Codable, Hashable {
    case good
    case risky
    case bad
}
